algorithm/classify.py

```diff
@@ -51,3 +59,45 @@ def get_redundant_data(self) -> dict[str:int]:
         Getter fn for redundant data
         """
         return self.redundant_data
+
+    def categorize_data(self):
+        """
+        Categorize the raw data into "name," "email," "birthday," and "cell_no" lists.
+        """
+
+        # Regex for cell numbers, birthdays, and emails
+        cell_no_regex = re.compile(r'^\+63-\d{10}$')
+        birthday_regex = re.compile(r'^\d{4}-\d{2}-\d{2}$')
+        email_regex = re.compile(r'^.+@.+\..{2,}$')
+
+        for data_set in self.raw_data:
+            for value in data_set:
+                value = value.strip()
```

Comment:
pahabol, i'm curious lang since values from data_set should be a string na, what does `value.strip()` do here? @marcusandrev